Sources/xcodes/main.swift

```diff
@@ -119,7 +119,7 @@ let install = Command(usage: "install <version>",
         installation = .latest
     } else if flags.getBool(name: "latest-prerelease") == true {
         installation = .latestPrerelease
-    } else if let url = flags.getString(name: "url"), let path = Path(url) {
+    } else if let url = flags.getString(name: "path"), let path = Path(url) {
         installation = .url(versionString, path)
```

Comment:
Could we rename `XcodeInstaller.InstallationType.url` to `XcodeInstaller.InstallationType.path` to match?